workshops/views.py

```diff
@@ -1017,14 +1046,50 @@ def event_delete(request, event_ident):
 def event_import(request):
     """Read tags from remote URL and return them as JSON.
 
-    This is used to read tags from workshop index page and then fill up fields
+    This is used to read tags from workshop website and then fill up fields
     on event_create form."""
+
+    url = request.POST['url'].strip()
     try:
-        url = request.POST['url']
-        translated_data = parse_tags_from_event_index(url)
-        return JsonResponse(translated_data)
-    except (KeyError, WrongEventURL):
-        raise SuspiciousOperation('Missing or wrong `url` POST parameter.')
+        # fetch page
+        response = requests.get(url)
+        response.raise_for_status()  # assert it's 200 OK
+        content = response.text
+
+        # find tags
+        tags = find_tags_on_event_website(content)
+
+        if 'slug' not in tags:
+            # there are no HTML tags, so let's try the old method
+            url = generate_url_to_event_index(url)
+
+            # fetch page
+            response = requests.get(url)
+
+            if response.status_code == 200:
+                # don't throw errors for pages we fall back to
+                content = response.text
+                tags = find_tags_on_event_index(content)
```

Comment:
I'd prefer [a more explicit versioning strategy][1], but this is enough for now.  Adding an explicit versioning strategy is something that gets more expensive with time, but it doesn't get *that* much more expensive ;).

[1]: https://github.com/swcarpentry/amy/pull/580#issuecomment-158125458